pkgs/development/python-modules/python-neutronclient/default.nix

```diff
@@ -0,0 +1,52 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, os-client-config
+, osc-lib
+, oslo-log
+, oslotest
+, hacking
+, openstackclient
+, setuptools
+, stestr
+}:
+
+buildPythonPackage rec {
+  pname = "python-neutronclient";
+  version = "11.3.1";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-U82ZI/Q6OwdypA41YfdGVa3IA4+QJhqz3gW2IR0S7cs=";
+  };
+
+  nativeBuildInputs = [
+    setuptools
+  ];
+
+  propagatedBuildInputs = [
+    osc-lib
+    openstackclient
+    os-client-config
+  ];
+
+  nativeCheckInputs = [
+    hacking
+    oslotest
+    oslo-log
```

Comment:
```suggestion
    oslo-log
    stestr
```

stestr is missing

Reply:
osprofiler is also a required python dependency needed for the tests
But looks like there is no package for it in NixOS: https://opendev.org/openstack/python-neutronclient/src/branch/master/test-requirements.txt#L8